### ISSUE TITLE
C++14 and OS X 10.11

### ISSUE THREAD

**Issue by 11011001** [Koderz/UE4RuntimeMeshComponent#2]
It looks like some of the latest commits have started to use some C++14 features, such as variable templates. Out of the box, C++14 is not supported by Unreal (at least on OS X) without rebuilding the build tools.

https://answers.unrealengine.com/questions/340300/can-my-game-be-configured-to-use-c14.html

**Comment by 11011001** [comment]
The error I got was: 

`error: variable templates are a C++14 extension [-Werror,-Wc++14-extensions]
FRuntimeMeshVertex<WantsPosition, WantsNormal, WantsTangent, WantsColor, NumWantedUVChannels, NormalTangentType, UVType>::TypeInfo;`

(I do not include the line number as I was reordering the declarations in RuntimeMeshGenericVertex.h to get it to compile on OS X)

**Comment by Koderz** [comment]
Well, I know I've only attempted to build the past few commits on my primary machine, but there shouldn't be anything in there that requires c++14, and I know I'm building with the stock build config of the engine, but it is MSVC. Unfortunately I don't really have access to an OS X machine right now so I'm limited on my testing abilities there.  The file that's from I don't think I've touched recently in master other than moving it into a submodule.
